keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -638,24 +640,107 @@ always dispatched immediately.  Following is how the other requests
 are dispatched at a given apiserver.
 
 The concurrency limit of an apiserver is divided among the non-exempt
-priority levels in proportion to their assured concurrency shares.
-This produces the assured concurrency value (ACV) for each non-exempt
-priority level:
+priority levels, and higher ones can do a limited amount of borrowing
+from lower ones.
+
+Non-exempt priority levels are ordered in a total order for the
+purpose of borrowing currently-unused concurrency.  The ordering is
+based (first) on increasing value of a spec field whose name is
+`priority` and whose value is constrained to lie in the range 0
+through 10000 inclusive and (second) on increasing name of the
+PriorityLevelConfiguration object.  Priority levels that appear later
+in the order are considered to have lower priority.
+
+Borrowing is done on the basis of the current situation, with no
+consideration of opportunity cost and no pre-emption when the
```

Comment:
I can imagine situations where this could be problematic. We may want to (let humans) disallow PLs with many slow requests from borrowing. (still reading so maybe you already solve this)

Reply:
No, the proposal does not address this issue.  I am not yet sure what I think about it.